src/main/java/it/gov/pagopa/payhub/activities/activity/classifications/ClearClassifyIufActivity.java

```diff
@@ -0,0 +1,14 @@
+package it.gov.pagopa.payhub.activities.activity.classifications;
+
+import it.gov.pagopa.payhub.activities.dto.classifications.ClassifyDTO;
+
+public interface ClearClassifyIufActivity {
+    /**
+     * deletion of a classification based on the provided dto parameter
+     *
+     * @param classifyDTO classification dto
+     * @return  boolean true for a successful deletion otherwise false
+     */
+    boolean deleteClassificationByIuf(ClassifyDTO classifyDTO);
```

Comment:
This is my mistake, to simplify the logic for deleting records. For deletion we need a unique identifier of the row, in this case the treasury ID. Surely we need the ID of an organization, let's rework this task.

Reply:
@marcomatteuccieng the event is the arrive of a payment reporting, which will provide us as input the IUF (maybe we have also the organizationId or the organizationFiscalCode, it depends on the payments reporting ingestion WF).
We need just to DELETE from the classification table the label TES_NO_MATCH related to the IUF (it could also not exists, we will just execute the DELETE without checking its existence)
The filter on the organizationId should be a reinforcement to avoid, if possible, the collision of IUF between organizations.

Reply:
I have reviewed the document, yes, with OrganizationID and IUF pair we can delete the record as the call is made from payments reporting.